test/Tmds.Ssh.Tests/SshServer.cs

```diff
@@ -247,6 +297,18 @@ public SshClientSettings CreateSshClientSettings(Action<SshClientSettings>? conf
     }
 }
 
+internal partial class Libc
```

Comment:
Can you use https://github.com/tmds/Tmds.ExecFunction to run the test that set environment variables in a separate process?

Reply:
That looks great, much better solution to using this, will try it out.